### PR TITLE
Update ISSUE_TRIAGE.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor grammar edits

### CODE REVIEW

File: ISSUE_TRIAGE.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+Triage helps ensure issues resolve quickly by:
+
+* Ensuring the issue's intent and purpose is conveyed precisely. This is necessary because it can be difficult for an issue to explain how an end user experiences a problem and what actions they took.
 * Giving a contributor the information they need before they commit to resolving an issue.
 * Lowering the issue count by preventing duplicate issues.
 * Streamlining the development process by preventing duplicate discussions.
 * If you don't have the knowledge or time to code, consider helping with triage. The community will thank you for saving them time by spending some of yours.
 
-**Simplified flowchart diagram of the issue triage process:**
+** Simplified flowchart diagram of the issue triage process:
```
Comment (marcusolsson) [inline ISSUE_TRIAGE.md]: I'm not sure this is what the author intended. The asterisks are used to make this sentence bold (although maybe it should be a heading?).

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Thanks for cleaning up these documents! 👍

**Comment by oddlittlebird** [comment]
I am quite sure it is not what the author intended, but it looks to me like
it should be a heading.

The content in this document is excellent, but it needs a thorough edit and
probably to be split into two or three documents:

* Grafana triage process, high level, include internal parts for external
contributor visibility
* how you can help triage
* other TBD

Anyway, all of that should be covered by the issue you are working on. This
is just a quick fix in the meantime.

File: ISSUE_TRIAGE.md (hunk 1 above)
Comment (marcusolsson) [reply ISSUE_TRIAGE.md]: Maybe `## Simplified flowchart diagram of the issue triage process`?

**Comment by oddlittlebird** [comment]
Marcus - 
The heading **1. Find uncategorized issues** is where the topic got very confusing for me. It was like another topic got spliced onto the first topic. Food for thought as you go through the contribution content. 

Also "ISSUE_TRIAGE" is a bad name for this topic. I expected a list of triaged issues. Something like TRIAGE_ISSUES or a a set of three docs similar to: CONTRIBUTE_CODE, CONTRIBUTE_DOCS, CONTRIBUTE_TRIAGE, would be much better.

File: ISSUE_TRIAGE.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Triage helps ensure issues resolve quickly by:
 * Streamlining the development process by preventing duplicate discussions.
 * If you don't have the knowledge or time to code, consider helping with triage. The community will thank you for saving them time by spending some of yours.
 
-** Simplified flowchart diagram of the issue triage process:
+** Simplified flowchart diagram of the issue triage process
```
Comment (marcusolsson) [inline ISSUE_TRIAGE.md]: This should either be:

Bold: 
`**Simplified flowchart diagram of the issue triage process**`

or

Heading:
`## Simplified flowchart diagram of the issue triage process`

**Review by marcusolsson — CHANGES_REQUESTED** [review]
There's definitely much work to be done here. Thank you for helping out!

**Comment by oddlittlebird** [comment]
Fixed the heading.

**Review by marcusolsson — APPROVED** [review]

**Comment by marcusolsson** [comment]
Thanks!

**Pull request merged**